src/Resolvers/Microsoft.DotNet.NativeWrapper/Interop.cs

```diff
@@ -40,6 +40,7 @@ private static void PreloadWindowsLibrary(string dllFileName)
         {
             string basePath = Path.GetDirectoryName(typeof(Interop).Assembly.Location);
             string architecture = IntPtr.Size == 8 ? "x64" : "x86";
+            architecture = RuntimeInformation.ProcessArchitecture == Architecture.Arm64 ? "arm64" : architecture;
```

Comment:
why not:
```suggestion
            string architecture = Enum.GetName(RuntimeInformation.ProcessArchitecture).ToLowerInvariant();
```

Reply:
For the sake of time, I decided not to consider the other code paths it could potentially go down if it were on a different arch. Though I suspect this to be the better way to go about it